src/convertToObject.js

```diff
@@ -7,6 +7,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.includes(':'))
+    .reduce((obj, item) => {
+      const [property, ...value] = item.split(':');
```

Comment:
The use of destructuring with `...value` is a good approach to handle potential CSS property values that contain colons. However, consider renaming the `value` variable to something more descriptive to improve code readability.